src/services/sanity.js

```diff
@@ -9,7 +9,9 @@ import {
     contentTypeConfig,
     DEFAULT_FIELDS,
     getFieldsForContentType,
-    filtersToGroq
+    filtersToGroq,
+    getUpcomingEventsTypes,
+    getNewReleasesTypes
```

Comment:
Pedantic, but trailing commas just mean the change set doesn't include extra lines. 
```suggestion
    getNewReleasesTypes,
```